MCLEN/iocBoot/iocMCLEN-IOC-01/st-motor-init.cmd

```diff
@@ -0,0 +1,28 @@
+## Initialises values at the motor level. Experience suggests the IOC doesn't do this automatically
+
+asynOctetConnect("MKINIT","$(ASERIAL)")
+
+## Some things that are helpful in development mode. Resetting, initialising. Probably don't want to do it in production
+#asynOctetWrite("MKINIT","$(MN)RS")
+#asynOctetWrite("MKINIT","$(MN)IN")
```

Comment:
Yes, I left them in because they can be quite useful when the motor gets in an odd state. I didn't want to remove them because I'd forget about them.